src/components/ComboBox.jsx

```diff
@@ -150,10 +150,7 @@ export default class ComboBox extends Component {
   static displayName = 'Belle ComboBox';
 
   static propTypes = {
-    children: React.PropTypes.oneOfType([
-      React.PropTypes.arrayOf(React.PropTypes.node),
-      React.PropTypes.node
-    ]),
+    children: React.PropTypes.oneOfType([React.PropTypes.array, React.PropTypes.object]),
```

Comment:
In this case you can simply say `React.PropTypes.object` because array is also an object and then the more detailed array check doesn't add value.